src/ui/processTree/darwinProcessTree.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { IProcess } from './processTree';
+import { BaseProcessTree } from './baseProcessTree';
+
+export class DarwinProcessTree extends BaseProcessTree {
+  /**
+   * @inheritdoc
+   */
+  protected createProcess() {
+    return this.spawn('/bin/ps', ['-wwxo', `pid=PID,ppid=PPID,comm=BINARY,command=COMMAND`]);
```

Comment:
This doesn't look necessary, the manpage says `When output is not to a terminal, an unlimited number of columns are always used`

Reply:
Ah, right, there was code in the v1 adapter that I assumed was to prevent truncation but looking at it again I think that was just to have a constant offset for slicing.